db/migrate/20150224142024_rename_question_group_to_survey_in_rapidfire_questions.rb

```diff
@@ -0,0 +1,5 @@
+class RenameQuestionGroupToSurveyInRapidfireQuestions < ActiveRecord::Migration
+  def change
+    rename_column :rapidfire_questions, :question_group_id, :survey_id
+  end
+end
```

Comment:
I am wondering if it is worth starting migrations from a clean slate, rather than doing all this. I am not sure, why anyone using this gem in a new project must have all this history generated. 

I think, we should have clean migration with one single rake task or something which migrates old one. thoughts?